backend/app/integrations/schemas.py

```
@@ -0,0 +1,198 @@
+"""
+Pydantic schemas for import operations.
+"""
+
+from datetime import datetime
+from enum import Enum
+from typing import Any, Dict, List, Optional
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)